nixos/modules/services/x11/window-managers/qtile.nix

```diff
@@ -26,7 +26,7 @@ in
       type = types.enum [ "x11" "wayland" ];
       default = "x11";
       description = ''
-          Backend to use in qtile: `x11` or `wayland`.
+          This option is deprecated. The qtile package now provides separate display sessions for both X11 and Wayland.
```

Comment:
We should add an assetion or probably even better use mkRemovedOption

Reply:
Much better indeed, fixed that now.